src/main/java/com/softserve/teamproject/service/UserProfileService.java

```diff
@@ -0,0 +1,16 @@
+package com.softserve.teamproject.service;
+
+import com.softserve.teamproject.entity.User;
+
+/**
+ * use for handle user profile information from/to database
+ */
+public interface UserProfileService {
+
+  /**
+   * get info from database about user by his NickName
+   * @param principalName user nick name
+   * @return user from database by his nickname
+   */
+  User getUserProfile(String principalName);
```

Comment:
Here and in the implementation class please start sentences with a capital letter